nodes/logic/loop_out.py

```diff
@@ -24,6 +24,12 @@
 from sverchok.core.update_system import make_tree_from_nodes, do_update
 from sverchok.data_structure import list_match_func
 
+def process_looped_nodes(node_list, tree_nodes):
+    for node_name in node_list:
+        try:
+            tree_nodes[node_name].process()
+        except Exception as e:
+            raise type(e)(str(e) + ' @ %s node' % node_name)
```

Comment:
I think this syntax could be used.
https://docs.python.org/3/tutorial/errors.html#exception-chaining

Reply:
That seems to change totally the exception and what we need is to add information to the existing exception https://stackoverflow.com/questions/6062576/adding-information-to-an-exception